index.js

```diff
@@ -16,10 +16,10 @@ app.use((req, res, next) => {
   next();
 });
 
-app.use('/', express.static(__dirname + '/client/build/'))
+app.use('/', express.static(__dirname + '/client/public'))
```

Comment:
This version of the server is intended to be used in production, it reads from the build as that will be where the build of your frontend will be.